concepts/documentation-strategy/webiste-engine-poc.md

```diff
@@ -0,0 +1,97 @@
+# [PoC] New engine for the Kyma open-source website
+
+## Context
+
+So far, the open-source project Kyma has been using https://kyma-project.io as its documentation webiste. The content has been pulled from the [`kyma` repository `/docs` folder](https://github.com/kyma-project/kyma/docs). The webiste's source is located in the [`webiste` repository](https://github.com/kyma-project/website/).
+
+As the ownership and responsibility for the `/webiste` have got lost over time, the project has become hard to maintain. Also, the main focus of the UI Kyma develpment team is on developing and maintaining Kyma Dashboard and not the documentation portal. The team's size and capacity is limited so the choice of tooling needs to be adjusted to these conditions.
+
+Additionally, [Kyma modularization](../modularization/) initiated changes to hosting all Kyma documentation in a central repository. The modular approach assumes that each module repository includes module documentation. As a result the content needs to be pulled from different repositories which is not an easy [`webiste`] feature to implement.
+
+The decision on the tooling choice may be influenced by the fact that the Kyma project is also a foundation of SAP BTP, Kyma runtime which is a part of SAP Business Technology Platform (BTP). The SAP BTP, Kyma runtime documentation is hosted on the SAP [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud).
+
+## Potential solutions
+
+To address the challenge, we should find and choose tooling to publish open-source Kyma documentation. Considering different options, take into account the following requirements:
+
+- user experience
+- concise and user-friendly domain
+- intuitive linking
+- search feature
+- right-side navigation displaying the document's headers (to avoid a really, really long left-side navigation node)
+- analytics feature
+
+### SAP Help Portal
+
+The SAP BTP, Kyma runtime documentation is hosted on the SAP [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud). One of the potential solutions is hosting open-source documentation using the same engine. Documentation on the Help Portal is created using the DITA CCMS.
+
+Potential setup:
+
+- open-source documentation generally available on the `Cloud Production` level ??
+- ??
+- internal documentation available on the `Internal draft` level ??
+
+#### Advantages:
+
+- The Help Portal provides the possibility of having different versions of documentation depending on the user type by ading flags and using different containers.
+- Having all Kyma documentation in Help Portal would enusre a unified user experince and only one entry point for Kyma users.
```

Comment:
maintenance in DITA CCMS doesn't exclusively mean publication on Help Portal. Though it definitely can.